install_config/aggregate_logging.adoc

```diff
@@ -111,72 +121,70 @@ API
 ----
 ====
 +
-Then, ensure the *logging-deployer* can edit the service. The following example
-uses the `logging` project. Be sure to use the project created above in the
-following example:
+Then, enable the *logging-deployer* service account to edit the project
+contents. By default service accounts are not allowed to do this.
+The following example uses the `logging` project. Be sure to use the
+project created above in the following example:
 +
 ====
 ----
 $ oc policy add-role-to-user edit \
-          system:serviceaccount:logging:logging-deployer
+            system:serviceaccount:logging:logging-deployer
+----
+====
+
+. Enable the Fluentd service account, which the deployer will create, that
+requires special privileges to operate Fluentd. Edit the security context:
++
+====
+----
+$ oc edit scc/privileged
+----
+====
++
+And add the following user to the file. The following example uses the `logging`
+project. Be sure to use the project created above in the following example:
++
+====
+----
+- system:serviceaccount:logging:aggregated-logging-fluentd
+----
+====
++
+Give the Fluentd service account permission to read labels from all pods. The
+following example uses the `logging` project. Be sure to use the project created
+above in the following example:
++
+====
+----
+$ oadm policy add-cluster-role-to-user cluster-reader \
+              system:serviceaccount:logging:aggregated-logging-fluentd
 ----
 ====
 
 == Deploying the EFK Stack
 
 The EFK stack is deployed link:../dev_guide/templates.html[using a template].
-Doing so ensures all future project configurations will feature the EFK stack.
```

Comment:
There is nothing about it in project config. Aggregated logs are not a project feature, they are a cluster feature.